.github/workflows/pfm-pr-merged.yml

```diff
@@ -0,0 +1,19 @@
+name: Mark Asana task complete on merge
+
+on:
+    pull_request:
+        types: [labeled, closed]
+
+jobs:
+    add-pr-merged-comment:
+        if: contains(join(github.event.pull_request.labels.*.name, ','), 'SBT-PFM') && (github.event.action == 'closed') && (github.event.pull_request.merged == true)
+        name: Process Pull Request, closing linked Asana task
+        runs-on: ubuntu-latest
+        steps:
+            - name: Add comment in Asana task
+              uses: duckduckgo/native-github-asana-sync@v1.1
```

Comment:
Any reason not to use the latest version - v1.5? Same applies to other version tags in this PR.